.github/workflows/performance-manager-ci.yaml

```diff
@@ -57,3 +57,31 @@ jobs:
         with:
           poetry-dir: performance_manager
       - run: poetry run pylint *.py lib/ tests/
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    needs: setup
+    env:
+      BOOTSTRAPPED: 1
+      DB_HOST: localhost
+      DB_PORT: 5432
+      DB_NAME: pm_test
+      DB_USER: postgres
+      DB_PASSWORD: postgres
+    services:
+      postgres:
+        image: postgres
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_PASSWORD: ${{env.DB_PASSWORD}}
+          POSTGRES_USER: ${{env.DB_USER}}
+          POSTGRES_DB: ${{env.DB_NAME}}
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/poetry_deps
+        with:
+          poetry-dir: performance_manager
+      - run: poetry run pytest
```

Comment:
add in step to run pytest. turns out we weren't doing this at all before?

in addition to running the tests, i added a postgres service that will be spun up in parallel that we can use to test against.